ioctl/readme/docgen.go

```diff
@@ -0,0 +1,206 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"path/filepath"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	"github.com/iotexproject/iotex-core/ioctl/cmd"
+)
+
+var ioctlPath = filepath.Join(os.Getenv("GOPATH"), "src", "github.com", "iotexproject", "iotex-core", "ioctl")
```

Comment:
not friendly for fork. consider something like https://coderwall.com/p/_fmbug/go-get-path-to-current-file

Reply:
how about filepath.Abs(".")?